### PR TITLE
chore(deps): Update `@aws-sdk` client libraries

### PR DESCRIPTION
**Pull request by AndesKrrrrrrrrrrr**
Using old versions causes issues with using login sessions.

## Summary by CodeRabbit

- **New Features**
	- Enhanced AWS SDK integration for retrieving SSM parameters, streamlining the parameter fetching process.
	- Improved error logging for AWS credentials issues, providing better context for troubleshooting.

- **Bug Fixes**
	- Updated multiple AWS SDK package versions to ensure compatibility and improve functionality.

- **Documentation**
	- Updated dependency versions in the project configuration for clarity and maintenance.

### CODE REVIEW

**Review by robertaandersen — APPROVED** [review]
LGTM

**Review by svanaeinars — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -69,12 +69,12 @@
...
     "@apollo/utils.keyvadapter": "3.0.0",
-    "@aws-sdk/client-s3": "3.662.0",
-    "@aws-sdk/client-sqs": "3.662.0",
-    "@aws-sdk/credential-provider-node": "3.662.0",
-    "@aws-sdk/lib-storage": "3.662.0",
-    "@aws-sdk/s3-presigned-post": "3.662.0",
-    "@aws-sdk/s3-request-presigner": "3.662.0",
+    "@aws-sdk/client-s3": "3.689.0",
+    "@aws-sdk/client-sqs": "3.689.0",
+    "@aws-sdk/credential-provider-node": "3.687.0",
```
Comment (baering) [inline package.json:74]: Any specific reason why this is `687` while the other changes are `689`?

Comment (AndesKrrrrrrrrrrr) [reply package.json:74]: No reason. I ran `yarn add $(jq -r '.dependencies | keys[]' package.json | grep '^@aws-sdk/')` to do the updates :shrug: 

Do you want the same version everywhere, or control which version we're using?

Comment (AndesKrrrrrrrrrrr) [reply package.json:74]: Set the same `3.677.0` for all. `@aws-sdk/credentials-provider-node` doesn't have a newer version.

**Review by baering — APPROVED** [review]

**Pull request merged**